board/piksiv3/rootfs-overlay/etc/init.d/S75dev_boot

```diff
@@ -50,22 +53,24 @@ start() {
   if [ -n "$dev_boot" ]; then
     # If dev_boot is set to something (not null), then we are in the dev
     # configuration
-    rm -f $output_path/piksi_firmware.elf
-    rm -f $output_path/piksi_fpga.bit
+    rm -f $lib_output_path/piksi_firmware.elf
+    rm -f $lib_output_path/piksi_fpga.bit
```

Comment:
I did this initially put pulled it out 71d594e211eb010952b404ba6d221d749bf892c9. I think the config should be left in place - this is optional, not mandatory. Don't want to destroy someone's config unless they want it destroyed by putting a config in their dev boot set.